zeppelin-integration/src/test/java/org/apache/zeppelin/integration/InterpreterModeActionsIT.java

```diff
@@ -203,12 +202,12 @@ void testGloballyAction() throws Exception {
       clickAndWait(By.xpath(
           "//div[@class='modal-dialog']//div[@class='bootstrap-dialog-footer-buttons']//button[contains(., 'OK')]"));
       clickAndWait(By.xpath("//a[@class='navbar-brand navbar-title'][contains(@href, '#/')]"));
-      interpreterModeActionsIT.logoutUser("admin");
+      logoutUser("admin");
```

Comment:
This class also contains the method `logoutUser`, we should only have the function once. My suggestion: not static in `AbstractZeppelinIT`.

Reply:
adopted